v3/sofle_pico/sofle_pico.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Sofle Pico",
```

Comment:
Sofle Pico doesnt exist in QMK master?

Reply:
How embarrassing!  That was another board in development. I removed it. Thank you for catching that!